helpers/README.md

```diff
@@ -42,6 +42,29 @@ module.exports = {
 };
 ```
 
+### Fixing violations
+
+An example of how to use the Node API to auto-fix violations in a string:
+
+```js
+const { sync } = require('markdownlint');
+const markdownlintRuleHelpers = require('markdownlint-rule-helpers');
+const markdownlintConfig = require('./.markdownlint.json');
+
+function fixMarkdownlintViolations(content) {
+  const fixResults = sync({
+    strings: { content },
+    config: markdownlintConfig,
+    resultVersion: 3,
+  });
+  const fixes = fixResults.content.filter((error) => error.fixInfo);
```

Comment:
Filtering should not be necessary, because I think violations without fixes are ignored. For simplicity, always directly calling the method should be fine.

Reply:
Simplified.

Reply:
I'm glad it's unnecessary. I was going to suggest that markdownlint needed a simpler API for fixing violations in a string (or file) without the need to have a bunch of logic/filtering for it. 

Calling `markdownlintSync` and then just `applyFixes` is much better.